### PR TITLE
Change to allow list in customize-tokens pages.

### PR DESCRIPTION
**Pull request by donwoods-okta**
## Description:
- Change to use "allow list" in customize-tokens pages.
- Change the name of the first parameter to getFilteredGroups, without the space (allowlist).
- Update examples (e.g. appuser.profile.groupallowlist).
- Other minor cleanup, including not directly related to the allow list change.
- Tied to the release that includes OKTA-341165 (error messages refer to the parameter by name).
- Should go with **2020.12.0** release

### Resolves:

* [OKTA-343920](https://oktainc.atlassian.net/browse/OKTA-343920)

### CODE REVIEW

**Comment by donwoods-okta** [comment]
Incidentally, I think after this PR, the only appearance of the string "white" in this entire repository is a single mention of "trailing white space", which I don't think has been deemed offensive yet.

**Comment by jakubvul-okta** [comment]
@donwoods-okta how will we know when to merge this?

**Comment by donwoods-okta** [comment]
@jakubvul-okta The associated code change is, as of OKTA-344317, behind the flag `MONTHLY_FLAG_2020_12`.  So the docs should be merged to coincide with that monthly release.

**Comment by jakubvul-okta** [comment]
This looks good to go once @johnhuang-okta approves (and 2020.12.0 is released, of course)

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/add-groups-claim-dynamic/index.md
Comment (johnhuang-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/add-groups-claim-dynamic/index.md:19]: After this change, we might need to change the archor name.

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/dynamic-allowlist-custom-as/index.md
Comment (johnhuang-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/dynamic-allowlist-custom-as/index.md:20]: Anchor name change?

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/add-groups-claim-dynamic/index.md
Comment (jakubvul-okta) [reply packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/add-groups-claim-dynamic/index.md:19]: Anchors are auto-generated by VuePress based on the heading.

**Review by johnhuang-okta — APPROVED** [review]

**Review by johnhuang-okta — COMMENTED** [review]
I took a second look and found some violations in packages/@okta/vuepress-site/.vuepress/nav/guides.js, which looks like owned by us as they are under customize-tokens-dynamic and customize-tokens-static. Can you also change them?

**Comment by donwoods-okta** [comment]
I'm not sure if that file is maintained by developers, since it's not under `vuepress-site/docs`.  @jakubvul-okta please advise.

**Comment by donwoods-okta** [comment]
I fixed the places @johnhuang-okta noted, and also a couple other pages I spotted that link to the main cluster.  PTAL.

**Pull request merged**
